src/mono/wasm/runtime/driver.c

```diff
@@ -315,6 +315,7 @@ mono_wasm_load_runtime (const char *managed_path, int enable_debugging)
 
 	//monoeg_g_setenv ("MONO_LOG_LEVEL", "debug", 0);
 	//monoeg_g_setenv ("MONO_LOG_MASK", "gc", 0);
+	monoeg_g_setenv ("COMPlus_DebugWriteToStdErr", "1", 0);
```

Comment:
Maybe add a comment explaining why we're setting this so that people don't have to git blame and then look at the PR

Reply:
Why are we even setting this in release build?

Reply:
We will need to add more to the build so that the driver build knows not to set in release.

Reply:
It should be a no-op on Release builds already since all of the `Debug` methods are under a `System.Diagnostics.Conditional("DEBUG")]`.

Reply:
The System.Diagnostics.DebugProvider.WriteCore () method is the only way to emit debug logging from inside corlib, and it requires this env var to be set to work. Note that stderr handling is still buggy somewhere, so the logs will still not show up.

Reply:
> It should be a no-op on Release builds

Will this be no-op on wasm runtime side?

Reply:
@marek-safar it doesn't really have anything to do with the wasm runtime, any Release build of the BCL will have the `Debug.*` methods as no-op.

Reply:
What I meant is that monoeg_g_setenv will add size cost on runtime if this is the only usage which might be possible

Reply:
There are multiple occurrences but the main case is that we also provide a public interface to `setenv` so environment variables can be set.

https://github.com/dotnet/runtime/blob/514455aafebc289916ee622608255279c7225a4f/src/mono/wasm/runtime/driver.c#L158

as well as https://github.com/dotnet/runtime/blob/514455aafebc289916ee622608255279c7225a4f/src/mono/wasm/runtime/driver.c#L158

Reply:
> Maybe add a comment explaining why we're setting this so that people don't have to git blame and then look at the PR

Addressed in recent commit